src/Platform/index.js

```diff
@@ -109,12 +114,32 @@ export default {
       return getOrSet('network', params)
     },
   },
-  Accesibility: {
+  Accessibility: {
     closedCaptions(params) {
       return getOrSet('closedCaptions', params)
     },
     voiceGuidance(params) {
       return getOrSet('voiceGuidance', params)
     },
   },
+  Personalization: {
+    setEntitlements(entitlements) {
```

Comment:
I prefer we use `params` as the argument for the methods, to stay consistent with the other methods

Naming wise, to stay in line with the rest of the API, I think we should rename this method to simply `entitlements` (i.e `Platform.Personalization.entitlements(params?)`. I assume that in practice this API should be able to also retrieve the entitlements from the platform / transport layer? So the getter / setter pattern would make sense.

Reply:
Yeah, I had exactly what you are suggesting initially.

I think we should have the conversation about methods that only go one way...

`setEntitlements` is set-only. you can't get them, since you're just passing them along to us so we know them. We're not the source of truth here, so it's not super useful to have a getter.

Reply:
> _I think we should have the conversation about methods that only go one way..._

For something that is one-way, I wonder if it makes more sense to use an event emitter pattern that's fire and forget. Leave it up to the transport layer to listen and act on events. Does that pattern already exist in the SDK?